crypto/signature_collector.go

```diff
@@ -95,17 +97,19 @@ func (sc *SignatureCollector) Accept(
 	}
 	sc.signature = string(buffer)
 	str, _ := ioutil.ReadAll(rawBody)
-	rawBody = bytes.NewReader(str)
+	rawBody = bytes.NewReader([]byte(internal.CanonicalizeAndTrim(string(str))))
```

Comment:
Do we always assume the body to be text?

Reply:
I think so, it seems that binary data is base64 encoded in PGP/MIME. So the whole content is text